src/components/tabController/PageCarousel.tsx

```diff
@@ -103,6 +101,7 @@ function PageCarousel({...props}) {
       scrollEventThrottle={16}
       contentOffset={initialOffset} // iOS only
       onLayout={scrollToInitial} // Android only
+      onMomentumScrollEnd={() => {}} // workaround for useAnimatedScrollHandler.onMomentumEnd not being called (https://github.com/software-mansion/react-native-reanimated/issues/2735)
```

Comment:
It'll override the `onMomentumScrollEnd` prop that the user might send, no?

Reply:
This is VERY strange, if the user sends `onMomentumScrollEnd` the workaround stops working (even though this is overriding it).

Reply:
Fixed by adding `useCallback`